src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,205 @@
+One column in the `*_tracksys-<label>_motion.tsv` file is intended to represent one data channel.
+The ordering of columns has to match the order of rows in the `*_channels.tsv` file for unambiguous assignment.
+All relevant metadata about a tracking systems is stored in accompanying sidecar `*_tracksys-<label>_motion.json` file.
+
+The source data from each tracking system in their original format, if different from `.tsv`,
+can be stored in the [`/sourcedata` directory](../common-principles.md#source-vs-raw-vs-derived-data).
+The original data format MAY hold more metadata than currently specified in the `*_motion.json` file.
+
+When multiple tracking systems are used to record motion or motion capture is used alongside the recording of other BIDS modalities, it is possible to temporally synchronize the recordings.
+A guideline to time synchronization between multiple modalities using recording onset and event time offset is described later in the specifications.
+To store the differences between recording onsets, [`scans.tsv`](../modality-agnostic-files.md#scans-file) files can be used.
+
+To store events which relate to a tracking system, it is recommended to use designated events files per tracking system.
+Such an events file name would include the `tracksys` entity and look like `sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv`.
+The onsets in the event file can be related to the starting time of the tracking system in the `scans.tsv` file.
+
+### Sidecar JSON (`*_motion.json`)
+
+#### Task information
+
+{{ MACROS___make_sidecar_table("motion.motionTaskInformation") }}
+
+#### Hardware information
+
+{{ MACROS___make_sidecar_table("motion.motionHardware") }}
+
+#### Insitution information
+
+{{ MACROS___make_sidecar_table("motion.motionInstitutionInformation") }}
+
+#### Motion specific fiels
+
+Motion specific fields MUST be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRequired") }}
+
+Motion specific fields SHOULD be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRecommended") }}
+
+Restricted keyword list for field `RotationRule`:
+
+| **Keyword** | **Description**                                                                                                                                                                 |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| left-hand   | Rotation is following the left-hand convention, such that the left thumb points to the positive end of the spatial axis, and the fingers curl along the orientation rotation.   |
+| right-hand  | Rotation is following the right-hand convention, such that the right thumb points to the positive end of the spatial axis, and the fingers curl along the orientation rotation. |
```

Comment:
Can we remove this table as well? `RotationOrder` is rendered in the spec like this: 
![grafik](https://user-images.githubusercontent.com/9084751/220084540-5a67c66b-86c0-4008-a7c0-949569ec3322.png)

Reply:
if some information would go missing, perhaps the metadata entry in the schema for `RotationRule` could be slightly extended to retain the information.